kolibri/plugins/management/assets/src/device_management/views/manage-permissions-page/index.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+
+  <subpage-container>
+
+      <auth-message v-if="!isSuperuser" authorizedRole="superuser" />
+
+      <div v-else>
+        <div>
+          <div class="one-half">
+            <h1>{{ $tr('devicePermissionsHeader') }}</h1>
+            <p>{{ $tr('devicePermissionsDescription') }}</p>
+          </div>
+          <div class="one-half">
+            <k-filter-textbox
```

Comment:
According to the filters spec, `Filter components should always be left aligned to the page.` The mockups show otherwise. I would suggest placing the filter below the description. @jtamiace @khangmach

Reply:
I take that back. So table layouts have a different layout spec for filters. They should be right aligned, below the title/Description. See examples in the [tables style guide doc](https://docs.google.com/document/d/11hAxq6tKRYYTD2pWZTb0cU1c2M_IannBo_N9cPB1eWo/edit#heading=h.wbyncvwo9sn3).

Reply:
@jb For now, lets keep the search filter right aligned, and further below the description. The device permissions mock is inaccurate in that the search filter is almost horizontally aligned to the page header and description. Sorry about that!

"Filter components should always be left aligned to the page" refers to filters that appear while navigating through pages (Learn/Topics). They are not referring to filters that exist within table patterns.